File: Bentzip/bentzip/lib/models/notice.dart
class Notice {
  String id;
  String title;
  String description;
  String date;

  Notice(this.id, this.title, this.description, this.date);

  factory Notice.fromJson(Map<String, dynamic> json) {
    return Notice(
        json["_id"], json["title"], json["description"], json["date"]);
  }
}

File: Bentzip/backend/models/notice.js
const mongoose = require("mongoose");

const Notice = mongoose.model("notice", new mongoose.Schema({
    title: {
        type: String,
        trim: true,
        required: true,
    },
    description: {
        type: String,
        trim: true,
        required: true,
    },
    school: {
        type: Number,
        trim: true,
        required: true,
    },
    date: {
        type: Date,
        default: Date.now()
    },
}));

module.exports = Notice;